### PR TITLE
App Icon Badging Csharp

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Raised PR for App Icon Badging Csharp

Please Review and Approve Changes 

Thanks!!

**Executed Build Pipeline :** 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5241311645

### CODE REVIEW

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Controllers/BotController.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        private readonly IBotFrameworkHttpAdapter Adapter;
+        private readonly IBot Bot;
+
+        public BotController(IBotFrameworkHttpAdapter adapter, IBot bot)
+        {
+            Adapter = adapter;
+            Bot = bot;
+        }
+
+        [HttpPost, HttpGet]
```
Comment (YunnyChung) [inline samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Controllers/BotController.cs:28]: I don't think we need Get here :)

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        /// </summary>
+        /// <param name="turnContext">The context object for this turn.</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects
+        /// or threads to receive notice of cancellation.</param>
+        /// <returns>A task that represents the work queued to execute.</returns>
+        private async Task HandleActions(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+        {
+            var action = JsonConvert.DeserializeObject<ActionBase>(turnContext.Activity.Value.ToString());
+
+            switch (action.Type)
```
Comment (YunnyChung) [inline samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs:98]: Just for my own understanding - 
(1) What other action.type are we planning to support here? 
(2) For the existing scenario (ex: targeted Meeting Notification), we support a different surface type (ex: Meeting Stage). is that scenario in a different bot?

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Controllers/BotController.cs (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Controllers/BotController.cs:28]: Yes, updated as per comment, will update on Teams Channel Thread once testing is done.

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs (hunk 2 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs:98]: Added Stage view as well like nodejs, once testing is done will update accordingly.
 Thanks.

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Manifest/manifest.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Meetings-App-Icon-Badging",
+    "full": "Show App Icon Badging In Teams Meetings"
+  },
+  "description": {
+    "short": "App Icon Badging In-Meeting Notification demo",
+    "full": "This sample app shows how to send app icon badging in Teams meetings using targeted notifications API."
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Manifest/manifest.json:23]: This sample app shows how to send app icon badging in Microsoft Teams meetings using targeted notifications API.

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Models/ActionBase.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
+﻿// <copyright file="ActionBase.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// </copyright>
+
+namespace InMeetingNotificationsBot.Models
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Models/ActionBase.cs:6]: Check namespace as per latest badge name.

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Pages/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Microsoft Teams Targeted Meeting Notification Sample</title>
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Pages/Shared/_Layout.cshtml:4]: Title looks wrong

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Program.cs [hunk 6]
```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Program.cs:3]: Remove one extra empty line

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Properties/launchSettings.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "EchoBot": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:3979;http://localhost:3978",
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Properties/launchSettings.json:22]: Should we add both port as 3978

File: samples/meetings-app-icon-badging/csharp/README.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+- If you are using Visual Studio
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples\meetings-app-icon-badging\csharp` folder
+  - Select `InMeetingNotifications.sln` file
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/README.md:64]: I think it should also update.

File: .github/workflows/build-complete-samples.yml [hunk 9]
```diff
@@ -436,6 +436,10 @@ jobs:
               name: 'bot-sso-adaptivecard'
               version: '6.0.x'
 
+            - project_path: 'samples/meetings-app-icon-badging/csharp/InMeetingNotifications/InMeetingNotificationsBot.csproj'
```
Comment (ChetanSharma-msft) [inline .github/workflows/build-complete-samples.yml:439]: I can't see deployment pipeline here.

Also, run build pipeline and add UI screenshot.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: .github/workflows/build-complete-samples.yml [hunk 10]
```diff
@@ -436,6 +436,10 @@ jobs:
...
+              name: 'meetings-app-icon-badging'
+              version: '6.0.x'
+
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply .github/workflows/build-complete-samples.yml:442]: Yes we have added nodejs sample as well.

File: README.md [hunk 11]
```diff
@@ -188,7 +188,8 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |28|Release Management     | This is a sample used to send workitem notification using Azure webhook.                                        |[View][release-management#cs]       |[View][release-management#nodejs]
 |29|Meeting Live Caption     | This is a sample meeting side panel application which demonstrates how to enable live caption in the meeting and using the CART link how to send caption in live meeting.                                       |[View][meetings-live-caption#cs]       |[View][meetings-live-caption#nodejs]
 |30|Anonymous user support| This sample app shows anonymous users support in teams meeting apps.|[View][app-anonymous-users#cs]       
-31|Link unfurling for 'Share to teams'|This sample demos the feature of link unfurling for Share to teams.||[View][link-unfurling-stt#nodejs]|
+|31|Link unfurling for 'Share to teams'|This sample demos the feature of link unfurling for Share to teams.||[View][link-unfurling-stt#nodejs]|
+|32|App Icon Badging In Teams Meetings|This sample demos the feature of sending app icon badging in teams meetings|[View][meetings-app-icon-badging#csharp]|[View][meetings-app-icon-badging#nodejs]|
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply README.md:192]: fixed

File: samples/meetings-app-icon-badging/csharp/README.md [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+Type `SendNotification` in bot chat to send In-Meeting notifications.
+
+**Notification On Tab Icon as 'Icon Badging':**
+
+![App Icon Badging](InMeetingNotifications/Images/3.AppIconBadging.png)
+
+
+## Further reading
+
+- [Enable app icon badging for your Teams app](https://review.learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/app-icon-badging-for-your-app?branch=pr-en-us-8495)
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/README.md:112]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/AdapterWithErrorHandler.cs [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+            OnTurnError = async (turnContext, exception) =>
+            {
+                // Log any leaked exception from the application.
+                // NOTE: In production environment, you should consider logging this to
+                // Azure Application Insights. Visit https://aka.ms/bottelemetry to see how
+                // to add telemetry capture to your bot.
+                logger.LogError(exception, $"[OnTurnError] unhandled error : {exception.Message}");
+
+                // Uncomment below commented line for local debugging.
+                // await turnContext.SendActivityAsync($"Sorry, it looks like something went wrong. Exception Caught: {exception.Message}");
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/AdapterWithErrorHandler.cs:25]: Verified, Is there

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Properties/launchSettings.json (hunk 7 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Properties/launchSettings.json:22]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Program.cs (hunk 6 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Program.cs:3]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Pages/Shared/_Layout.cshtml (hunk 5 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Pages/Shared/_Layout.cshtml:4]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs [hunk 14]
```diff
@@ -0,0 +1,189 @@
...
+                Value = new TargetedMeetingNotificationValue()
+                {
+                    Recipients = recipients,
+                    Surfaces = new List<Surface>()
+                    {
+                        new MeetingTabIconSurface()
+                    },
+                }
+            };
+            return notification;
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs:186]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Manifest/manifest.json (hunk 3 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Manifest/manifest.json:23]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs [hunk 15]
```diff
@@ -0,0 +1,189 @@
+﻿// <copyright file="InMeetingNotificationsBot.cs" company="Microsoft">
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs:1]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs [hunk 16]
```diff
@@ -0,0 +1,189 @@
...
+
+            // Get card from result
+            AdaptiveCard card = result.Card;
+
+            var adaptiveCardAttachment = new Attachment
+            {
+                ContentType = AdaptiveCard.ContentType,
+                Content = card,
+            };
+            return adaptiveCardAttachment;
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Bots/InMeetingNotificationsBot.cs:164]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Controllers/BotController.cs [hunk 17]
```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+
+
+namespace InMeetingNotificationsBot
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Controllers/BotController.cs:10]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Models/ActionBase.cs (hunk 4 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Models/ActionBase.cs:6]: fixed

File: samples/meetings-app-icon-badging/csharp/README.md (hunk 8 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/README.md:64]: fixed

File: samples/meetings-app-icon-badging/csharp/InMeetingNotifications.sln [hunk 18]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/InMeetingNotifications.sln:3]: fixed

**Comment by Mohammed-MSFT** [comment]
Executed Build Pipeline : https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5357679397

File: README.md [hunk 19]
```diff
@@ -188,7 +188,8 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |28|Release Management     | This is a sample used to send workitem notification using Azure webhook.                                        |[View][release-management#cs]       |[View][release-management#nodejs]
 |29|Meeting Live Caption     | This is a sample meeting side panel application which demonstrates how to enable live caption in the meeting and using the CART link how to send caption in live meeting.                                       |[View][meetings-live-caption#cs]       |[View][meetings-live-caption#nodejs]
 |30|Anonymous user support| This sample app shows anonymous users support in teams meeting apps.|[View][app-anonymous-users#cs]       
-31|Link unfurling for 'Share to teams'|This sample demos the feature of link unfurling for Share to teams.||[View][link-unfurling-stt#nodejs]|
+|31|Link unfurling for 'Share to teams'|This sample demos the feature of link unfurling for Share to teams.||[View][link-unfurling-stt#nodejs]|
+|32|App Icon Badging In Microsoft Teams Meetings|This sample demos the feature of sending app icon badging in teams meetings|[View][meetings-app-icon-badging#csharp]|[View][meetings-app-icon-badging#nodejs]|
```
Comment (ChetanSharma-msft) [inline README.md:192]: This sample app shows the feature of sending app icon badging in Teams' meetings.

File: samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AdapterWithErrorHandler.cs [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+            OnTurnError = async (turnContext, exception) =>
+            {
+                // Log any leaked exception from the application.
+                // NOTE: In production environment, you should consider logging this to
+                // Azure Application Insights. Visit https://aka.ms/bottelemetry to see how
+                // to add telemetry capture to your bot.
+                logger.LogError(exception, $"[OnTurnError] unhandled error : {exception.Message}");
+
+                // Uncomment below commented line for local debugging.
+                // await turnContext.SendActivityAsync($"Sorry, it looks like something went wrong. Exception Caught: {exception.Message}");
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AdapterWithErrorHandler.cs:25]: Have you added the note in the readme to uncomment it for debugging?

File: samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Manifest/manifest.json [hunk 21]
```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.15",
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Manifest/manifest.json:3]: Can't we use 1.16?

File: samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Manifest/manifest.json [hunk 22]
```diff
@@ -0,0 +1,83 @@
...
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Meetings-App-Icon-Badging",
+    "full": "Show App Icon Badging In Teams Meetings"
+  },
+  "description": {
+    "short": "App Icon Badging In-Meeting Notification demo",
+    "full": "This sample app shows how to send app icon badging in Microsoft Teams meetings using targeted notifications API.."
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Manifest/manifest.json:23]: This sample app shows how to send app icon badging in Microsoft Teams meetings using targeted notifications API.

File: samples/meetings-app-icon-badging/csharp/README.md [hunk 23]
```diff
@@ -0,0 +1,116 @@
...
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+- If you are using Visual Studio
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples\meetings-app-icon-badging\csharp` folder
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/README.md:63]: please check if it's correct?
samples\meetings-app-icon-badging\csharp

should it be like: samples/meetings-app-icon-badging/csharp

File: samples/meetings-app-icon-badging/csharp/README.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (ChetanSharma-msft) [reply samples/meetings-app-icon-badging/csharp/README.md:112]: @Mohammed-MSFT - This is the review doc URL, we need to update it with production URL.

File: samples/meetings-app-icon-badging/csharp/README.md [hunk 24]
```diff
@@ -0,0 +1,116 @@
...
+![App Icon Badging](AppIconBadgingInMeetings/Images/3.AppIconBadging.png)
+
+
+## Further reading
+
+- [Enable app icon badging for your Teams app](https://review.learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/app-icon-badging-for-your-app?branch=pr-en-us-8495)
+
+- [Grant RSC permissions to your app](https://learn.microsoft.com/en-us/microsoftteams/platform/graph-api/rsc/grant-resource-specific-consent#install-your-app-in-a-team-or-chat)
+
+<img src="https://pnptelemetry.azurewebsites.net/microsoft-teams-samples/samples/app-icon-badging/csharp" />
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/README.md:116]: app-icon-badging/csharp
Looks like this path is wrong.

File: samples/meetings-app-icon-badging/csharp/assets/sample.json [hunk 25]
```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-meetings-app-icon-badging-csharp",
+    "source": "officeDev",
+    "title": "Meetings-App-Icon-Badging",
+    "shortDescription": "Illustrates how to implement app icon badging \"in-meeting notification\" using \"targeted meeting notification\" API",
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/assets/sample.json:6]: Illustrates how to => This sample shows how to..............

File: samples/meetings-app-icon-badging/csharp/assets/sample.json [hunk 26]
```diff
@@ -0,0 +1,68 @@
...
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "bot"
```
Comment (ChetanSharma-msft) [inline samples/meetings-app-icon-badging/csharp/assets/sample.json:31]: Looks wrong, add all capability

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added review comments.

File: .github/workflows/build-complete-samples.yml [hunk 27]
```diff
@@ -436,6 +436,10 @@ jobs:
               name: 'bot-sso-adaptivecard'
               version: '6.0.x'
 
+            - project_path: 'samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AppIconBadgingInMeetings.csproj'
```
Comment (ChetanSharma-msft) [inline .github/workflows/build-complete-samples.yml:439]: @Mohammed-MSFT - Kindly add deploy pipeline as well.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added one more comment.

File: README.md (hunk 19 above)
Comment (Mohammed-MSFT) [reply README.md:192]: fixed

File: samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AdapterWithErrorHandler.cs (hunk 20 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AdapterWithErrorHandler.cs:25]: yes

File: samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Manifest/manifest.json (hunk 21 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Manifest/manifest.json:3]: yes we can

File: samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Manifest/manifest.json (hunk 22 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/Manifest/manifest.json:23]: fixed

File: samples/meetings-app-icon-badging/csharp/README.md (hunk 23 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/README.md:63]: fixed

File: samples/meetings-app-icon-badging/csharp/README.md (hunk 24 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/README.md:116]: fixed

File: samples/meetings-app-icon-badging/csharp/assets/sample.json (hunk 25 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/assets/sample.json:6]: fixed

File: samples/meetings-app-icon-badging/csharp/assets/sample.json (hunk 26 above)
Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/csharp/assets/sample.json:31]: fixed

File: .github/workflows/build-complete-samples.yml (hunk 27 above)
Comment (Mohammed-MSFT) [reply .github/workflows/build-complete-samples.yml:439]: done

**Comment by Mohammed-MSFT** [comment]
Executed Build Pipe Line : https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5398239303

**Comment by ChetanSharma-msft** [comment]
image

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5409638458

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**